Clear doxygen warnings from global_cycle.fd

Part of #191 

There are many doxygen warnings from global_cycle.fd. ;-)

@GeorgeGayno-NOAA and @Hang-Lei-NOAA can you tackle these? I will work on other UFS_UTILS documentation warnings, and will pitch in with global_cycle if I finish first.

You will need to coordinate so you don't both do the same file at the same time. 

Here's some of the warnings, the rest can be seen in the CI:

```
Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/machine.f90:16: warning: Member kind_dbl_prec (variable) of module machine is not documented.
524
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/machine.f90:15: warning: Member kind_evod (variable) of module machine is not documented.
525
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/machine.f90:12: warning: Member kind_io4 (variable) of module machine is not documented.
526
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/machine.f90:13: warning: Member kind_io8 (variable) of module machine is not documented.
527
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/machine.f90:14: warning: Member kind_ior (variable) of module machine is not documented.
528
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:52: warning: Member kpdais (variable) of module sfccyc_module is not documented.
529
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:67: warning: Member kpdalb_0 (variable) of module sfccyc_module is not documented.
530
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:68: warning: Member kpdalb_1 (variable) of module sfccyc_module is not documented.
531
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:69: warning: Member kpdalf (variable) of module sfccyc_module is not documented.
532
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:52: warning: Member kpdgla (variable) of module sfccyc_module is not documented.
533
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:52: warning: Member kpdplr (variable) of module sfccyc_module is not documented.
534
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:52: warning: Member kpdsno (variable) of module sfccyc_module is not documented.
535
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:52: warning: Member kpdtg3 (variable) of module sfccyc_module is not documented.
536
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:52: warning: Member kpdtsf (variable) of module sfccyc_module is not documented.
537
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:52: warning: Member kpdwet (variable) of module sfccyc_module is not documented.
538
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:52: warning: Member kpdzor (variable) of module sfccyc_module is not documented.
539
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:71: warning: Member mdata (variable) of module sfccyc_module is not documented.
540
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:73: warning: Member soil_type_landice (variable) of module sfccyc_module is not documented.
541
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:72: warning: Member veg_type_landice (variable) of module sfccyc_module is not documented.
542
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:71: warning: Member xdata (variable) of module sfccyc_module is not documented.
543
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:71: warning: Member ydata (variable) of module sfccyc_module is not documented.
544
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'slmask' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
545
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fntsfa' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
546
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnweta' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
547
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnsnoa' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
548
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnzora' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
549
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnalba' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
550
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnaisa' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
551
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fntg3a' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
552
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnscva' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
553
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnsmca' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
554
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnstca' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
555
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnacna' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
556
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnvega' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
557
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnveta' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
558
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnsota' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
559
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnvmna' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
560
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnvmxa' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
561
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnslpa' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
562
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'fnabsa' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
563
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'tsfanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
564
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'wetanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
565
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'snoanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
566
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'zoranl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
567
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'albanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
568
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'aisanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
569
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'tg3anl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
570
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'cvanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
571
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'cvbanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
572
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'cvtanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
573
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'smcanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
574
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'stcanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
575
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'slianl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
576
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'scvanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
577
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'acnanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
578
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'veganl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
579
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'vetanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
580
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'sotanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
581
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'alfanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
582
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'tsfan0' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
583
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'vmnanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
584
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'vmxanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
585
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'slpanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
586
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'absanl' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
587
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdtsf' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
588
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdwet' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
589
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdsno' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
590
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdsnd' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
591
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdzor' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
592
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdalb' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
593
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdais' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
594
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdtg3' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
595
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdscv' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
596
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdacn' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
597
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdsmc' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
598
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdstc' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
599
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdveg' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
600
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kprvet' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
601
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdsot' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
602
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdalf' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
603
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdvmn' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
604
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdvmx' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
605
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdslp' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
606
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'kpdabs' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
607
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irttsf' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
608
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtwet' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
609
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtsno' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
610
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtzor' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
611
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtalb' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
612
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtais' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
613
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irttg3' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
614
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtscv' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
615
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtacn' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
616
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtsmc' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
617
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtstc' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
618
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtveg' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
619
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtvet' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
620
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtsot' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
621
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtalf' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
622
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtvmn' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
623
/Users/runner/work/UFS_UTILS/UFS_UTILS/ufs_utils/sorc/global_cycle.fd/sfcsub.F:3921: warning: argument 'irtvmx' of command @param is not found in the argument list of analy(lugb, iy, im, id, ih, real(kind=kind_io8) fh, len, lsoil)
```

--- a/sorc/global_cycle.fd/machine.f90
+++ b/sorc/global_cycle.fd/machine.f90
@@ -8,17 +8,8 @@ MODULE MACHINE
 
  IMPLICIT NONE
  SAVE
-!  Machine dependant constants
- integer, parameter :: kind_io4  = 4
- integer, parameter :: kind_io8  = 8
- integer, parameter :: kind_ior = 8
- integer, parameter :: kind_evod = 8
- integer, parameter :: kind_dbl_prec = 8
- integer, parameter :: kind_rad  = selected_real_kind(13,60) !< the '60' maps to 64-bit real
- integer, parameter :: kind_phys = selected_real_kind(13,60) !< the '60' maps to 64-bit real
- integer, parameter :: kind_REAL = 8 !< used in cmp_comm
- integer, parameter :: kind_INTEGER = 4 !< -,,-
-!
- real(kind=kind_evod), parameter :: mprec = 1.e-12 !< machine precision to restrict dep
-
+ integer, parameter :: kind_io4  = 4 !< Kind type for 4-byte float point
+                                     !! variables.
+ integer, parameter :: kind_io8  = 8 !< Kind type for 8-byte float point
+                                     !! variables.
  END MODULE MACHINE

--- a/sorc/global_cycle.fd/sfcsub.F
+++ b/sorc/global_cycle.fd/sfcsub.F
@@ -6850,8 +6850,8 @@
 !! @param[in] data The input data.
 !! @param[in] imax "i" dimension of input grid.
 !! @param[in] jmax "j" dimension of input grid.
-!! @param[out] rlnout Latitudes on input data grid.
-!! @param[out] rltout Longitudes on input data grid.
+!! @param[out] rlnout Longitudes on input data grid.
+!! @param[out] rltout Latitudes on input data grid.
 !! @param[out] lmask True, when input data has a mask.
 !! @param[out] rslmsk The mask of the input data grid.
 !! @param[in] gaus Is high-resolution mask on a gaussian grid.
@@ -7343,22 +7343,22 @@
       return
       end
 
-!> ???
+!> Interpolation from lat/lon or gaussian grid to a lat/lon grid.
 !!
-!! @param[in] gauin
-!! @param[in] imxin
-!! @param[in] jmxin
-!! @param[in] regout
-!! @param[in] imxout
-!! @param[in] jmxout
-!! @param[in] wlon
-!! @param[in] rnlat
-!! @param[in] rlnout
-!! @param[in] rltout
-!! @param[in] gaus
-!! @param[in] blno
-!! @param[in] blto
-!! @author M. Iredell, xuli, Hang Lei, George Gayno
+!! @param[in] gauin Input data.
+!! @param[in] imxin 'i' dimension of input data.
+!! @param[in] jmxin 'j' dimension of input data.
+!! @param[out] regout Output data.
+!! @param[in] imxout 'i' dimension of output data.
+!! @param[in] jmxout 'j' dimension of output data.
+!! @param[in] wlon Longitude of west boundary of input data.
+!! @param[in] rnlat Latitude of north row of input data.
+!! @param[in] rlnout Longitudes on output data grid.
+!! @param[in] rltout Latitudes on output data grid.
+!! @param[in] gaus Is input data on gaussian grid?
+!! @param[in] blno Corner point longitude of input data.
+!! @param[in] blto Corner point latitude of input data.
+!! @author Shrinivas Moorthi
       subroutine ga2la(gauin,imxin,jmxin,regout,imxout,jmxout,
      &                 wlon,rnlat,rlnout,rltout,gaus,blno, blto)
       use machine , only : kind_io8,kind_io4

--- a/sorc/global_cycle.fd/sfcsub.F
+++ b/sorc/global_cycle.fd/sfcsub.F
@@ -49,11 +49,39 @@
 !
 !  grib code for each parameter - used in subroutines sfccycle and setrmsk.
 !
-      integer kpdtsf,kpdwet,kpdsno,kpdzor,kpdais,kpdtg3,kpdplr,kpdgla,
-     &        kpdmxi,kpdscv,kpdsmc,kpdoro,kpdmsk,kpdstc,kpdacn,kpdveg,
-     &        kpdvet,kpdsot
-     &,       kpdvmn,kpdvmx,kpdslp,kpdabs
-     &,       kpdsnd, kpdabs_0, kpdabs_1, kpdalb(4)
+      integer kpdtsf !< GRIB1 parameter number of skin temperature/SST.
+      integer kpdwet !< GRIB1 parameter number of soil wetness.
+      integer kpdsno !< GRIB1 parameter number of liquid equivalent snow
+                     !! depth.
+      integer kpdzor !< GRIB1 parameter number of physical snow depth.
+      integer kpdais !< GRIB1 parameter number of roughness length.
+      integer kpdtg3 !< GRIB1 parameter number of soil substrate
+                     !! temperature.
+      integer kpdplr !< GRIB1 parameter number of plant resistance.
+      integer kpdgla !< GRIB1 parameter number of glacial ice.
+      integer kpdmxi !< GRIB1 parameter number of maximum ice extent.
+      integer kpdscv !< GRIB1 parameter number of snow cover.
+      integer kpdsmc !< GRIB1 parameter number of soil moisture.
+      integer kpdoro !< GRIB1 parameter number of orography.
+      integer kpdmsk !< GRIB1 parameter number of land mask.
+      integer kpdstc !< GRIB1 parameter number of soil temperature.
+      integer kpdacn !< GRIB1 parameter number of sea ice concentration.
+      integer kpdveg !< GRIB1 parameter number of vegetation greenness.
+      integer kpdvet !< GRIB1 parameter number of vegetation type.
+      integer kpdsot !< GRIB1 parameter number of soil type.
+      integer kpdvmn !< GRIB1 parameter number of minimum vegetation
+                     !! greenness.
+      integer kpdvmx !< GRIB1 parameter number of maximum vegetation
+                     !! greenness.
+      integer kpdslp !< GRIB1 parameter number of slope type.
+      integer kpdabs !< GRIB1 parameter number of maximum snow albedo.
+      integer kpdsnd !< GRIB1 parameter number of physical snow depth.
+      integer kpdabs_0 !< GRIB1 parameter number of legacy maximum snow
+                       !! albedo.
+      integer kpdabs_1 !< GRIB1 parameter number of modis-based 
+                       !! maximum snow albedo.
+      integer kpdalb(4) !< GRIB1 parameter number of 4-component
+                        !! snow-free albedo.
       parameter(kpdtsf=11,  kpdwet=86, kpdsno=65,  kpdzor=83,
 !    1          kpdalb=84,  kpdais=91, kpdtg3=11,  kpdplr=224,
      1          kpdais=91,  kpdtg3=11, kpdplr=224,
@@ -64,14 +92,22 @@
      &          kpdvet=225, kpdsot=224,kpdabs_1=159,
      &          kpdsnd=66 )
 !
-      integer, parameter :: kpdalb_0(4)=(/212,215,213,216/)
-      integer, parameter :: kpdalb_1(4)=(/189,190,191,192/)
-      integer, parameter :: kpdalf(2)=(/214,217/)
-!
-      integer, parameter :: xdata=5000, ydata=2500, mdata=xdata*ydata
-      integer            :: veg_type_landice
-      integer            :: soil_type_landice
-!
+      integer, parameter :: kpdalb_0(4)=(/212,215,213,216/) !< GRIB1 parameter
+                                                         !! numbers for brigleb
+                                                         !! snow-free albedo.
+      integer, parameter :: kpdalb_1(4)=(/189,190,191,192/) !< GRIB1 parameter
+                                                         !! numbers for modis 
+                                                         !! snow-free albedo.
+      integer, parameter :: kpdalf(2)=(/214,217/) !< GRIB1 parameter numbers of
+                                                !! fraction for strongly/weakly
+                                                !! zenith angle dependent albedo.
+      integer, parameter :: xdata=5000 !< Maximum 'i' dimension of input data.
+      integer, parameter :: ydata=2500 !< Maximum 'j' dimension of input data.
+      integer, parameter :: mdata=xdata*ydata !< Maximum number of input data points.
+      integer            :: veg_type_landice !< Vegetation type category at permanent
+                                             !! land-ice points.
+      integer            :: soil_type_landice !< Soil type category at permanent
+                                             !! land-ice points.
       end module sfccyc_module
 
 !> Surface cycling driver routine. Update 'first guess' surface

